src/include/add-on/add-on-workflow.rb

```diff
@@ -1027,6 +1027,9 @@ def RunAddOnMainDialog(enable_back, enable_next, enable_abort, back_button, next
           Wizard.SetTitleIcon("yast-addon")
           ret2 = RunWizard()
 
+          break if ret2 == :back
+          return ret2 if ret2 == :abort
```

Comment:
NP: maybe 
```suggestion
          return :abort if ret2 == :abort
```

is a bit more readable...

Reply:
For sure! Thanks!